src/neuroconv/datainterfaces/behavior/deeplabcut/_dlc_utils.py

```diff
@@ -58,14 +58,11 @@ def _get_movie_timestamps(movie_file, VARIABILITYBOUND=1000, infer_timestamps=Tr
     n_frames = int(reader.get(cv2.CAP_PROP_FRAME_COUNT))
     fps = reader.get(cv2.CAP_PROP_FPS)
 
+    warnings.warn("Inferring timestamps from video. This might take a while (to speed up, set timestamps)")
```

Comment:
From memory this takes minutes for hours long videos. Is my memory wrong here? If it is not, I rather would not have this warning by default. I am fine with adding something to skip this relative expensive process but I don't like to use warnings as tool to inform about optimizations. We can do that in the docstring.

Reply:
I would suggest to keep the warning. Several minutes, up to 15-20 for me (also, depending on video frame rate, and support from which video is read) can feel very wrong if a user is just linking an external video. At least it did to me. Could go in doctstring, but I would personally find it harder to get this piece of info. 

But I do not encounter this issue myself anymore, totally up to you guys to decide on this!

Reply:
Ok! I see the point, thank you for explaining further. Actually yes, having the message on the loading bar would be much better! I'll take the morning out here

Reply:
Thanks!